### PR TITLE
added Shipping Zone resource

### PR DESCRIPTION
**Pull request by kazinov**

### CODE REVIEW

File: test/shipping_zone_test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+var helper = require("./common.js");
+helper.setObject("shipping_zone");
+
+var Resource = helper.resource();
+
+helper.nock(helper.test_shop)
+  .get('/admin/shipping_zones.json')
+  .reply(200, helper.load("all"), {
```
Comment (lpinca) [inline test/shipping_zone_test.js]: Save the fixture somewhere, so we can reuse it later and remove the third argument.

File: test/shipping_zone_test.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    status: '200 OK'
+  });
+
+describe('Shipping zone', function() {
+  var site = helper.endpoint;
+  var resource = new Resource(site);
+
+  it('should get all shipping zones', function(done) {
+    resource.all(function(err, res){
+      res.should.not.be.empty;
```
Comment (lpinca) [inline test/shipping_zone_test.js]: Use the previously saved fixture, e.g.`res.should.be.eql(fixture.shipping_zones)`;

Comment (kazinov) [inline test/shipping_zone_test.js]: @lpinca, fixed

Comment (lpinca) [inline test/shipping_zone_test.js]: Please also remove `res.should.not.be.empty;`, replace it with `if (err) return done(err);`

**Comment by kazinov** [comment]
@lpinca fixed

**Comment by lpinca** [comment]
Awesome, thank you.

**Pull request merged**